config/manager/manager.yaml

```diff
@@ -6,14 +6,42 @@ metadata:
   name: system
 ---
 apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nutanix-endpoint
+  namespace: capi-nutanix-system
+data:
+  prismCentral: |-
+    {
```

Comment:
since this is a json, should we be ideally adding a version format telling the schema version?

Reply:
also, we are going away from strong typing to a loose json format.

Reply:
This is only for manager component. Workload cluster specs are unchanged.